contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -2834,7 +2862,23 @@ static void bbf_ProcessUtility(PlannedStmt *pstmt,
 					if (HeapTupleIsValid(tuple))
 						roleform = (Form_pg_authid) GETSTRUCT(tuple);
 					else
-						continue;
+					{
+						/* Supplied login name might be in windows format i.e, domain\user form */
+						tuple = get_roleform_ext(role_name);
```

Comment:
Please update the PR description as this also includes DROP LOGIN.

Reply:
Also, shouldn't we handle the case when the customer directly drops the login@domain format? By the way, it should be domain/login instead of domain/user in the comments.

Reply:
> shouldn't we handle the case when the customer directly drops the login@domain format?
We will not allow it, user should use same format everywhere. In future version, we may add support for it based on feedback.